ballerina/sql_client.bal

```diff
@@ -24,28 +24,41 @@ public client class SQLClient {
     private sql:ParameterizedQuery tableName;
     private map<FieldMetadata> fieldMetadata;
     private string[] keyFields;
+    private map<JoinMetadata> joinMetadata;
 
-    public function init(string entityName, sql:ParameterizedQuery tableName, map<FieldMetadata> fieldMetadata, string[] keyFields, sql:Client dbClient) returns error? {
+    public function init(string entityName, sql:ParameterizedQuery tableName, map<FieldMetadata> fieldMetadata, string[] keyFields,
+                         sql:Client dbClient, map<JoinMetadata> joinMetadata = {}) returns error? {
```

Comment:
Shall we rearrange the param order like,

```ballerina
    public function init(sql:Client dbClient, string entityName, sql:ParameterizedQuery tableName, string[] keyFields, map<FieldMetadata> fieldMetadata, map<JoinMetadata> joinMetadata = {}) returns error? {
```

Reply:
Fixedin [8a80d26](https://github.com/ballerina-platform/module-ballerina-persist/pull/11/commits/8a80d263be98cd46f659f4c7ee8429cce0848163)